ops/README.md

```diff
@@ -197,3 +188,58 @@ Create the `snapshot` and `version` repositories, with:
 The @OTA-Bot GitHub user should have write access to all three (declarations, snapshots, versions) repositories, so it can publish data, create issues, and publish dataset releases.
 
 Each instance should have a responsible entity, which we currently model as a [“team” in the @OpenTermsArchive](https://github.com/orgs/OpenTermsArchive/teams) GitHub organisation. Each team has write access to the three repositories, and @OTA-Bot should be added to that team along with the human maintainers.
+
+## Development
+
+Note that the VM needs to be started before running any commands with `vagrant up`.
+
+If you’re on an Apple Silicon machine or want to use Docker instead of VirtualBox, type `vagrant up --provider=docker`.
+### Additional dependencies
+
+To test the changes without impacting the production server, a Vagrantfile is provided to test the changes locally in a virtual machine. <br />[Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.vagrantup.com/docs/installation/) or [Docker](https://docs.docker.com/get-docker/) are therefore required.
+
+### SSH with vagrant
+
+In order to be able to connect to vagrant through SSH, and deploy with ansible, you need to create a custom SSH key by using:
+
+```
+ssh-keygen -t rsa -f ~/.ssh/ota-vagrant -q -N ""
+```
+
+#### On a Mac with an Apple Silicon processor
+
+VirtualBox is not compatible with Apple Silicon (M1…) processors. You will thus need to use the Docker provider.
+
+**Setup**
+
+Install the `docker` plugin
+
+```
+vagrant plugin install docker
+```
+
+**Launch**
+
+```
+vagrant up --provider=docker
+```
+
+You can then deploy the code to the running machine with
+
+```
+ansible-playbook ops/site.yml
+```
+
+:warning: When deploying on architecture `aarch64`, [as MongoDB and Chromium install does not work](https://github.com/ambanum/OpenTermsArchive/issues/743), they are skipped in infrastructure installation process. This means you cannot test MongoDB storage with Vagrant on ARM architecture.
+
+**Connect to the running machine**
+```
+vagrant ssh
+```
+
+**Stop and destroy**
+
+```
+vagrant halt # stop machine
+vagrant destroy -f # remove machine
+```
```

Comment:
> I'm not sure how relevant this is in our own documentation.

Reply:
I believe referring to another doc for simple tasks as this ones are often very painful for new users (and i was one). That's why I put them here.